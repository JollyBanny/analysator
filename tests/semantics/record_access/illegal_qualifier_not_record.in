var rec : integer;

begin
    write(rec.y);
end.