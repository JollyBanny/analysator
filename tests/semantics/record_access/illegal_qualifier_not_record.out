[Line: 4; Column: 11]	illegal qualifier
